README.md

```diff
@@ -0,0 +1,92 @@
+# Event Store
```

Comment:
This should be client specific... it can link to the OSS server repo for the rest of the information